# AnVIL Urban Genomics: Identifying population structure among feral pigeon populations

[![Render Bookdown and Coursera](https://github.com/jhudsl/OTTR_Template/actions/workflows/render-all.yml/badge.svg)](https://github.com/jhudsl/OTTR_Template/actions/workflows/render-all.yml)

This course was created from [the AnVIL Template](https://github.com/jhudsl/AnVIL_bookdown_style), which is based on [the OTTR Template](https://github.com/jhudsl/OTTR_Template)

You can see the rendered course material here: [https://hutchdatascience.org/AnVIL_Urban_Genomics_PCA/](https://hutchdatascience.org/AnVIL_Urban_Genomics_PCA/)

## About this course

This book is part of a series of books for the Genomic Data Science Analysis, Visualization, and Informatics Lab-space (AnVIL) of the National Human Genome Research Institute (NHGRI). This course introduces genetics concepts like DNA variants and population level variation and dynamics. This course also has a module guiding students to work with SNP data from a subsample of 50 feral pigeons (collected in 6 different Northeastern US cities) that were captured and sampled in Carlen and Munshi-South’s study. Students will calculate PCA and create plots to visualize the population structure among these pigeons using the R programming language.

## Learning Objectives

This activity addresses the following Core Concepts and Competencies as stated in the [Genetics Core Competencies](https://genetics-gsa.org/education/genetics-learning-framework/):

- Core Competency: Critique large data sets and use bioinformatics to assess genetics data.
- Core Competency: Generate and interpret graphs displaying experimental results.
- Evolution and Population Genetics: Explain how natural selection and genetic drift can affect the elimination, maintenance or increase in frequency of various types of alleles (e.g. dominant, recessive, deleterious, beneficial) in a population.

This activity also addresses the following core competency from the Bioinformatics core competencies for undergraduate life sciences education from [NIBLSE](https://journals.plos.org/plosone/article/figure?id=10.1371/journal.pone.0196878.t002):

- Use bioinformatics tools to examine complex biological problems in evolution, information flow, and other important areas of biology.
- Use command-line bioinformatics tools and write simple computer scripts.

This course's background lectures will teach learners to

- Learn what genomics and genetics are.
- Learn what variants, SNVs, and SNPs are.
- Learn how variants can impact an organism’s phenotype.
- Learn how DNA variants are stored in datafiles.
- Learn what urbanization is and how it affects the abiotic and biotic environments.
- Understand the difference between the urban facilitation and urban fragmentation hypothesis.
- Learn what population structure, gene flow, and genetic drift are.
- Understand how gene flow and genetic drift behave under the urban facilitation and urban fragmentation hypotheses.
- Learn what principal component analysis is and why it is useful in bioinformatics.
- Understand how PCs are calculated
- Learn about the basic natural history of rock pigeons.
- Understand the difference between wild, domesticated, and feral pigeons.
- Understand why pigeons have been successful in the urban environment.

## Encountering problems?

If you are encountering any problems with this course, please file a GitHub issue or contact us at {Some email or web address with a contact form}.

<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />All materials in this course are licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a> unless noted otherwise.
